### DISCUSSION TITLE
Disable some variants and extra props.

### DISCUSSION THREAD

**Discussion by santomegonzalo** [GeekyAnts/NativeBase#5127] [General]
Hey everybody!
It is great to have a library with so many options and properties, but we want to disable some of the variants for example on Inputs. I do not want anybody to use variants that our design team is not supporting. 
Is there any way to do this?

**Comment by akinzalowevidation** [comment]
We're in the same boat. There are variants that we do _not_ want to support. So far we've just been setting their theme to the default variant's theme.
